tests/unit/testNumberService.unitTest.ts

```diff
@@ -0,0 +1,269 @@
+import {TestNumberService} from "../../src/services/TestNumberService";
+import {TestNumber} from "../../src/models/TestNumber";
+import {DynamoDBService} from "../../src/services/DynamoDBService";
+import {HTTPResponse} from "../../src/utils/HTTPResponse";
+jest.mock("../../src/services/DynamoDBService");
+
+describe("TestNumberService", () => {
+    const testNumberService = new TestNumberService(new DynamoDBService());
+    beforeEach(() => {
+        jest.restoreAllMocks();
+        jest.resetModules();
+    });
+    describe("createNextTestNumberObject", () => {
+        context("corner cases", () => {
+            it("should return proper nextTestNumber", () => {
```

Comment:
missing async

Reply:
No I didn't - createNextTestNumberObject isn't asynchronous, so not required.